src/content/docs/opentelemetry/best-practices/opentelemetry-otlp.mdx

```diff
@@ -246,7 +247,7 @@ Additionally, you should enable compression to reduce payload size and limit the
 
 The mechanism to configure the endpoint will vary, but OpenTelemetry language SDKs generally support setting the `OTEL_EXPORTER_OTLP_COMPRESSION=gzip` environment variable (see [OpenTelemetry docs](https://opentelemetry.io/docs/specs/otel/protocol/exporter/) for more info).
 
-If using the collector, `gzip` is the default compression, but `zstd` can be optionally configured.
+If using the Collector, `gzip` is the default compression, but `zstd` can be optionally configured.
```

Comment:
I would write this to avoid the passive voice:
`If using the Collector, `gzip` is the default compression, but you can optionally configure `zstd`.`